packaging/packager_deb.sh

```diff
@@ -68,4 +69,4 @@ chmod 0700 /tmp/runtime-root
 cd /usr/lib/python3/dist-packages/guibot/tests
 LIBPATH=".." COVERAGE="python3-coverage" sh coverage_analysis.sh
 
-exit 0
+exit 0
```

Comment:
You have removed the last empty line here

Reply:
This is still not addressed it seems

Reply:
I have remove it in the latest commit!